### PR TITLE
Add sharifelgamal to minikube-maintainers

### PR DESCRIPTION
**Pull request by tstromberg**
@sharifelgamal has been helping with minikube maintenance tasks, and recently joined the Kubernetes org

Thanks!

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -1824,6 +1824,7 @@ teams:
     - ethernetdan
     - jimmidyson
     - tstromberg
+    - sharifelgamal
```
Comment (nikhita) [inline config/kubernetes/org.yaml]: this needs to be alphabetical :)

Comment (tstromberg) [reply config/kubernetes/org.yaml]: Done, and thank you!

**Comment by nikhita** [comment]
@tstromberg could you also squash the commits, otherwise lgtm :+1:

**Comment by tstromberg** [comment]
> @tstromberg could you also squash the commits, otherwise lgtm

Done.

**Pull request merged**